config/scheduler/settings.ini

```diff
@@ -446,6 +446,19 @@ pa.scheduler.core.tenant.filter=false
 # It is possible to allow both named domain and empty domain using the syntax pa.scheduler.allowed.domains=,domain1,domain2
 # pa.scheduler.allowed.domains=domain1,domain2
 
+# Regular expression used to control the username format when logging in. Always enabled to prevent injection attacks on 3rd-party authentication like LDAP
+pa.scheduler.username.regexp=^[A-Za-z0-9_\\-@.]+$
+
+# Enable password strength check, used when a user is created or modified
+pa.scheduler.password.strength.enable=false
+
+# Regular expression used to control the password strength (default is 8 to 32 characters with at least a uppercase letter, a lowercase letter, a digit and a special character)
+pa.scheduler.password.strength.regexp=^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[*.!@$%^&(){}\\[\\]:;<>,.?/~_+\\-=|]).{8,32}$
+
+# Textual error message when password strength is not met
+pa.scheduler.password.strength.message=The password must be 8 to 32 characters long and include at least one uppercase letter, one lowercase letter, one digit, and one symbol.
```

Comment:
Why have the error message configurable ?

Reply:
Ah ok to adapt it depending on the chosen strength regex

Reply:
Clearly a bit more understandable for users :-)